card/mod/05_standard/set/type/signup.rb

```diff
@@ -22,29 +22,29 @@ def default_new_args args
   end
 
   view :new do |args|
-    #FIXME - make more use of standard new view?
+    # FIXME - make more use of standard new view?
```

Comment:
Annotation keywords should be all upper case, followed by a colon and a space, then a note describing the problem.